src/bson/ext/int.cr

```diff
@@ -1,3 +1,27 @@
+struct Int8
```

Comment:
I think this is useless, `to_canonical_extjson` will only ever get called for supported bson types (Int32 & Int64)

Reply:
Fixed in following commit : 988b1c24c7e520fec575478e0ba307c40b779a3e